src/components/EventCard/EventCard.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import HeartIconEmpty from "../Icons/HeartIconEmpty";
+import HeartIconFilled from "../Icons/HeartIconFilled";
+import styles from "./EventCard.module.css";
+
+type EventCardProps = {
+  imgSrc: string;
+  name: string;
+  date: string;
+  time: string;
+  location: string;
+  price: string;
+};
+
+export default function EventCard({
+  imgSrc,
+  name,
+  date,
+  time,
+  location,
+  price,
+}: EventCardProps): JSX.Element {
+  const [isButtonClicked, setIsButtonClicked] = useState(false);
```

Comment:
I think this should be coming from a prop as boolean and `onClick` should be a prop as well to pass a function to that button :) that way this would be a completely controlled component :)